setup.cfg

```diff
@@ -9,3 +9,8 @@ directory=build/reports/coverage
 
 [coverage:xml]
 output=build/reports/coverage.xml
+
+[flake8]
```

Comment:
We have moved codestyle configuration to `.prospector.yaml`, since that's the tool used by codacy.
In that file we specify `max-line-length: 140`.

Is there maybe some README or other docs that needs to have a section on codestyle tooling?

Reply:
> Is there maybe some README or other docs that needs to have a section on codestyle tooling?

When I think about it, let's fix that in a separate PR.

Reply:
Aha, I wasn't aware of that. My editor uses flake8 by default and thus complained a bit, but it can probably be persuaded to use prospector. I'll remove this

Reply:
I think line length should be 120 since that fits in the github PR view, whereas 140 doesn't